.travis_scripts/validate.sh

```diff
@@ -9,4 +9,4 @@ EOF
     false
 )
 
-sbt +mimaReportBinaryIssues +test docs/test
+sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues test docs/test
```

Comment:
No need to cross build/test as a Travis instance is spawn per each Scala version